src/types/events.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Related plugin events which are dispatched externally through the player.
+ */
+const RelatedEvent = {
```

Comment:
this is a model not a type so i think it should not be in types/ directory , i suggest to create a new file called related-events

Reply:
i've moved it to another folder in this PR:
https://github.com/kaltura/playkit-js-related/pull/37/files#diff-2a70666c28d275ea3d772c26f80af76782534e0feae34c3b483e409dfc87ee9f